src/protocol/eas/eas.ts

```diff
@@ -0,0 +1,459 @@
+      'timestamp',
+      'multiTimestamp',
+      'revokeOffchain',
+      'multiRevokeOffChain',
+    ].includes(decoded.name);
+  } catch (err) {
+    console.error('Error in detect function:', err);
+    return false;
+  }
+};
+
+// Contextualize for mined txs
+export const generate = (transaction: Transaction): Transaction => {
+  const decoded = decodeTransactionInput(transaction.input, ABIs.EAS);
+
+  switch (decoded.name) {
+    case 'attest': {
+      const { schema, data } = decoded.args[0];
+      const { recipient } = data;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          schema: {
+            type: 'emphasis',
+            value: schema,
+          },
+          recipient: {
+            type: 'address',
+            value: recipient,
+          },
+          attested: {
+            type: 'contextAction',
+            value: 'ATTESTED',
+          },
+        },
+        summaries: {
+          category: 'PROTOCOL_1',
+          en: {
+            title: 'EAS',
+            default: recipient
+              ? '[[from]] [[attested]] to [[recipient]] with schema [[schema]]'
+              : '[[from]] [[attested]] with schema [[schema]]',
+          },
+        },
+      };
+      return transaction;
+    }
+
+    case 'attestByDelegation': {
+      const { schema, attester, data } = decoded.args[0];
+      const { recipient } = data;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          attester: {
+            type: 'address',
+            value: attester,
+          },
+          schema: {
+            type: 'emphasis',
+            value: schema,
+          },
+          recipient: {
+            type: 'address',
+            value: recipient,
+          },
+          attested: {
+            type: 'contextAction',
+            value: 'ATTESTED',
+          },
+        },
+        summaries: {
+          category: 'PROTOCOL_1',
+          en: {
+            title: 'EAS',
+            default: recipient
+              ? '[[attester]] [[attested]] to [[recipient]] with schema [[schema]] by delegation via [[from]]'
+              : '[[attester]] [[attested]] with schema [[schema]] by delegation via [[from]]',
+          },
+        },
+      };
+      return transaction;
+    }
+
+    case 'multiAttest': {
+      const schemas = decoded.args[0].length;
+      const count = decoded.args[0].map((v) => v.data).flat().length;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          schemas: {
+            type: 'emphasis',
+            value: schemas.toString(),
+          },
+          count: {
+            type: 'emphasis',
+            value: count.toString(),
+          },
```

Comment:
@datadanne We could explore including the unit for this number, which is `times` here, as part of the variable payload here for use in other languages for internationalization. We'd probably have to update the types to allow this.
```suggestion
          count: {
            type: 'emphasis',
            value: count.toString(),
            unit: 'times',
          },
```

Reply:
Ah this would be nice! A bit out of scope for this PR I think but it feels like it would be worth adding :+1: